jangaroo-browser/src/main/joo/js/AbortSignal.as

```diff
@@ -0,0 +1,27 @@
+package js {
+
+/**
+ * A signal object that allows you to communicate with a DOM request (such as a Fetch) and abort it if required via an
+ * {@link AbortController} object.
```

Comment:
Unfortunately, there is no @link in ActionScript.
Our convention is to use something like a footnote style:

    ... →<code>link-ref|link-text</code> ...

    @see link-ref [link-text]

Reply:
I wouldn't use this technique to link String or Array. Too many @see to such well-known classes don't really improve readability.